### PR TITLE
home-page

### PR DESCRIPTION
**Pull request by prakashsakari**
Added home page for the right fit store, kindly review my PR. Thank you

snip
snip-24
u

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    <header class="heading d-flex grow1-shrink1-basisauto align-center">
+        <div class="heading-title-icon d-flex align-center">
+            <img class="icon mr-1 border-radius-50" src="assets/The Right Fit-logos.jpeg" alt="lightbul">
+            <h1 class="heading-title">
+                <a class="link" href="/">The Right Fit</a>
+            </h1>
+        </div>
+        <div class="search-box-container relative">
+            <input class="search-box padding-all-8 border-radius-4" type="text" placeholder="Search">
+            <img src="/assets/outline_search_black_24dp.png" alt="" class="search-icon absolute left-0 top-0">
```
Comment (Naveenchandar) [inline index.html]: Please provide "alt" description for accessibility.

Comment (prakashsakari) [reply index.html]: thanks, updated!

**Review by Naveenchandar — APPROVED** [review]
Looks good

File: main.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    background-repeat: no-repeat;
+}
+
+.new-arrival-section {
+    gap: 5rem;
+}
+
+.new-arrival-banner-container{
+    width: 43vw;
+    height: 17vw;
```
Comment (aniketsingh13) [inline main.css:125]: any reason for not using rem

**Review by aniketsingh13 — APPROVED** [review]
look good to me

File: main.css (hunk 2 above)
Comment (prakashsakari) [reply main.css:125]: I wanted somewhere between 40 to 45% of the screen width as the width of my container and somewhere between 15% to 20% as the height, that's the reason.

**Pull request merged**